components/Datepicker/style.module.scss

```diff
@@ -0,0 +1,58 @@
+@import "styles/index";
+
+.mdl-container-wrapper {
+	
+	:global(.react-datepicker) {
+		box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.15);
+		border-radius: 2px;
+		border: none;
+		background-color: $color-m-white;
+		font-family: $font-family;
+		font-size: $base-font-size;
+	}
+
+	:global(.react-datepicker__header) {
+		background-color: $color-m-white;
+		border-bottom: none;
+	}
+
+	:global(.react-datepicker__current-month) {
+		font-weight: normal;
+		font-size: $base-font-size;
+		margin: 2.5px 10.9px 10px 2.6px;
+		color: $color-night;
+	}
+
+	:global(.react-datepicker__day) {
+		margin: 10px 10.8px 9px 8.7px;
+		color: $color-deepgray;
+	}
+
+	:global(.react-datepicker__day:hover) {
+		background-color: $color-gray;
+		color: $color-night;
+		border-radius: 12px;
+		font-weight: bold;
+	}
+
+	:global(.react-datepicker__day-names) {
```

Comment:
Tightening can be done to another level here. If above is changed to `&__day {`, then this one can be moved under that one and then shortened to `&-names {`.